content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -214,6 +215,14 @@ To request a larger volume for a PVC, edit the PVC object and specify a larger
 size. This triggers expansion of the volume that backs the underlying `PersistentVolume`. A
 new `PersistentVolume` is never created to satisfy the claim. Instead, an existing volume is resized.
 
+#### CSI Volume expansion
+
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}}
+
+CSI volume expansion requires enabling `ExpandCSIVolumes` feature gate and
+running external resize controller. Please see [external resize controller(https://github.com/kubernetes-csi/external-resizer/)] for more details.
```

Comment:
oh actually this is more of a detail for the storage driver, not the kubernetes user right?  You can try following the pattern of the [csi topology docs](https://github.com/kubernetes/website/pull/12889/files#diff-568e6d19b36c4c9d5cda7af639a2c8daR154)

Reply:
I prefer style used here actually.  I am posting screenshot of how both docs look below:

![Screenshot from 2019-03-18 11-23-45](https://user-images.githubusercontent.com/278/54541658-b4ccdb80-4970-11e9-965c-ca85c8e48d7c.png)

![Screenshot from 2019-03-18 11-23-22](https://user-images.githubusercontent.com/278/54541664-b72f3580-4970-11e9-8ef7-3bc3f7767c5b.png)

Reply:
I have fixed the link render btw.